pdns/dnsdistdist/meson.build

```diff
@@ -8,7 +8,7 @@ project(
   default_options: [
     'buildtype=debugoptimized',
     'warning_level=2',          # TODO Move this to 3 to enable -Wpedantic
-    'cpp_std=c++17',
+    'cpp_std=c++20',
```

Comment:
while we're here, I'd love to switch this to gnu++20, which improves portability (at least to Haiku, perhaps also other OSes)

Reply:
As long as clang does the right thing...

Reply:
I believe it does, but it deserves the same amount of testing you did for this PR. We can also do it separately. None of our currently shipping targets care about the difference anyway.